paper-spinner-lite.html

```diff
@@ -48,7 +48,7 @@
 -->
 
 <dom-module id="paper-spinner-lite">
-  <template strip-whitespace>
+  <template>
```

Comment:
isn't `strip-whitespace` better because it's faster? I remember we were adding it because it's a performance gain